sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -372,8 +372,8 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
     struct PathAppendDataResult
     {
-      Azure::Core::Nullable<std::string> ContentMD5;
-      Azure::Core::Nullable<std::string> ContentCrc64;
+      Azure::Core::Nullable<ContentHash> ContentMD5;
+      Azure::Core::Nullable<ContentHash> ContentCrc64;
```

Comment:
Only one result will be returned from server side, why do we need two separate properties?